Add Indonesian Manage Compute Resources page

phanama opened the pull request:
This PR addresses #15341 
Requesting reviews
/assign @irvifa

phanama commented:
/assign @irvifa

danninov commented:
/assign @danninov

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+* `spec.containers[].resources.requests.ephemeral-storage`
+
+Limit dan request untuk `ephemeral-storage` diukur dalam satuan _bytes_. Kamu dapat menyatakan
+penyimpanan dalam bilangan bulat biasa, atau sebagai _fixed-point integer_ menggunakan satu dari
```
penyimpanan dalam bilangan integer biasa

phanama replied:
Let's use bilangan bulat instead for consistency

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+
+### Bagaimana Pod-Pod dengan request ephemeral-storage dijadwalkan
+
+Saat kamu membuat sebuah Pod, Kubernetes scheduler memilih sebuah Node di mana Pod
```
Saat kamu membuat sebuah Pod, Kubernetes _scheduler_

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+tersebut akan dijalankan. Setiap Node memiliki jumlah maksimum penyimpanan lokal sementara yang dapat disediakan.
+Untuk lebih lanjut, lihat ["Hal-hal yang dapat dialokasikan Node"](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable).
+
+Scheduler memastikan bahwa jumlah dari request-request sumber daya dari Container-Container yang dijadwalkan lebih sedikit dari kapasitas Node.
```
_Scheduler_ memastikan bahwa jumlah dari _request-request_ sumber daya dari Container-Container yang dijadwalkan lebih rendah dari kapasitas Node.

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+Saat kamu membuat sebuah Pod, Kubernetes scheduler akan memilih sebuah Node
+untuk Pod tersebut untuk dijalankan. Setiap Node memiliki kapasitas maksimum
+untuk setiap jenis sumber daya: jumlah CPU dan memori yang dapat disediakan
+oleh Node tersebut untuk Pod-Pod. Scheduler memastikan bahwa, untuk setiap
```
_Scheduler_ memastikan bahwa untuk setiap

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+untuk setiap jenis sumber daya: jumlah CPU dan memori yang dapat disediakan
+oleh Node tersebut untuk Pod-Pod. Scheduler memastikan bahwa, untuk setiap
+jenis sumber daya, jumlah semua request sumber daya dari Container-Container
+yang dijadwalkan lebih sedikit dari kapasitas Node tersebut. Perlu dicatat
```
yang dijadwalkan lebih rendah dari kapasitas Node tersebut.

phanama replied:
I'll use "lebih kecil" instead

phanama replied:
I'll use lebih kecil instead

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+
+### Bagaimana Pod-Pod dengan limit ephemeral-storage dijalankan
+
+Untuk isolasi pada tingkat kontainer, jika lapisan yang dapat ditulis dari sebuah Container dan penggunaan log melebihi limit penyimpanannya, maka Pod tersebut akan dipindahkan. Untuk isolasi pada tingkat Pod, jika jumlah dari penyimpanan lokal sementara dari semua Container dan juga volume emptyDir milik Pod melebihi limit, maka Pod teresebut akan dipindahkan.
```
Untuk isolasi pada tingkat Container, jika lapisan yang dapat ditulis dari sebuah Container dan penggunaan log melebihi limit penyimpanannya, maka Pod tersebut akan dipindahkan. Untuk isolasi pada tingkat Pod, jika jumlah dari penyimpanan lokal sementara dari semua Container dan juga volume emptyDir milik Pod melebihi limit, maka Pod tersebut akan dipindahkan.

phanama replied:
Here's the original english passage:

```
For container-level isolation, if a Container’s writable. . .
```

I think the first "container" in "container-level" refers to the container as in container the technology, not the Container API resource, so I left it as "kontainer" instead of using the capitalized untranslated "Container".
What do you guys think? @danninov @girikuncoro @irvifa @wahyuoi

wahyuoi replied:
+1 for kontainer, it's not referring to Kubernetes object

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+mereka secara otomatis telah dibangun dengan _project quotas_ yang
+telah diaktifkan.
+
+Sistem berkas _ext4fs_ harus dibangun denga  mengaktifkan _quotas_,
```
harus dibangun dengan

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+##### Sumber daya lainnya
+
+Untuk menyatakan sebuah sumber daya yang diperluas tingkat Node, operator klaster
+dapat mengirimkan permintaan HTTP `PATCH` ke API server untuk memerinci kuantitas
```
untuk menentukan kuantitas

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+Kolom `status.allocatable` diperbarui secara otomatis dengan sumber daya baru
+tersebut secara _asynchrounous_ oleh Kubelet. Perlu dicatat bahwa karena scheduler
+menggunakan nilai `status.allocatable` milik Node saat mengevaluasi muat atau tidaknya
+Pod, mungkin ada waktu tunda pendek antara melakukan `PATCH` terhadap kapasitas Node
```
mungkin ada waktu jeda

danninov commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+`3Ki`. Contoh jumlah yang _tidak valid_ adalah `0.5` dan `1500m`.
+
+{{< note >}}
+Sumber daya yang diperluas menggantikan Opaque Integer Resource.
```
Sumber daya yang diperluas menggantikan _Opaque Integer Resource_.

phanama replied:
I believe Opaque Integer Resource refers to an old API object (now removed), so I didn't italicize it. What do you think? @danninov

wahyuoi replied:
It was API object, but already replaced with Extended resources

irvifa commented:
Hi @phanama Any update on this?

phanama commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+dapat secara opsional memerinci seberapa banyak CPU dan memori (RAM) yang dibutuhkan
+oleh setiap Container. Saat Container-Container memerinci _request_ (permintaan) sumber daya,
+scheduler dapat membuat keputusan yang lebih baik mengenai Node mana yang akan dipilih
+untuk menaruh Pod-Pod. Dan saat Container-Container memerinci _limit_ (batas) sumber
```
Here, I am trying to use limit without italic, so I use (batas) to help the readers on occurences of limit for the rest of the translation.

phanama replied:
Just found out that limit is an indonesian word! Haha 😂 
I'll de-italicize it and use this technique on "request" instead.

phanama commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+* `spec.containers[].resources.requests.memory`
+* `spec.containers[].resources.requests.hugepages-<size>`
+
+Walaupun `requests` dan `limits` hanya dapat diperinci pada Container individual, akan
```
They are field names, so I use `requests` and `limits` as an emphasis to help the readers!

phanama commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+* `spec.containers[].resources.requests.hugepages-<size>`
+
+Walaupun `requests` dan `limits` hanya dapat diperinci pada Container individual, akan
+lebih mudah untuk membahas tentang request dan limit sumber daya dari Pod. Sebuah
```
I have italicized "request" and put a (permintaan) on its first occurence here: https://github.com/kubernetes/website/pull/18468/files#diff-76b56d02c3d293cc69f7b43df7e748ebR15

So I think it'll help the reader to not italicize the rest of the "request" words.
What do you think?

phanama replied:
@danninov

wahyuoi replied:
similar with abbreviation, isn't it? 
but I never know about this approach for italicized word. is there any reference @phanama ?

phanama commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+
+## Bagaimana Pod-Pod dengan request sumber daya dijadwalkan
+
+Saat kamu membuat sebuah Pod, Kubernetes scheduler akan memilih sebuah Node
```
We might translate "scheduler" to "penjadwal" or, italicize it.

But, it is the name of a Kubernetes component:
https://kubernetes.io/docs/concepts/scheduling/kube-scheduler/

Do you guys think we should translate it, italicize it, or just leave it as is? @danninov @girikuncoro @irvifa @wahyuoi @qbl 
I think we can leave it as is for our translations to avoid confusion to the readers?

wahyuoi replied:
agree with @phanama
leave it as is when it's Kubernetes component

phanama commented on content/id/docs/concepts/configuration/manage-compute-resources-container.md:
```diff
@@ -0,0 +1,632 @@
+  Periode kuota bawaan adalah 100ms. Resolusi minimum dari kuota CPU adalah 1 milidetik.
+  {{</ note >}}
+
+- `spec.containers[].resources.limits.memory` diubah menjadi sebuah bilangan, dan
```
Anddd... I just found out that Integer is a registered indonesian word haha
https://kbbi.kemdikbud.go.id/entri/integer
.
.
I'll translate it to bilangan bulat instead.

phanama commented:
Thanks for the review! @danninov 

I've replaced all the "rinci" to "tentukan" and fixed the typos in the https://github.com/kubernetes/website/pull/18468/commits/5e9dfb8ae2960263bbf4ae9571c3c38866409b76 commit.

Regarding "request", I've left a comment here: https://github.com/kubernetes/website/pull/18468#discussion_r365350517

Regarding "scheduler", I've left a comment for discussion here:
https://github.com/kubernetes/website/pull/18468#discussion_r365351689

And here's some worth some discussion:
https://github.com/kubernetes/website/pull/18468#discussion_r365354586
https://github.com/kubernetes/website/pull/18468#discussion_r365355424

phanama commented:
Thanks for the review! @danninov 

I've replaced all the "rinci" to "tentukan" and fixed the typos in the https://github.com/kubernetes/website/commit/e16ce57e7ef329fff26dc986f2f5b3178178dc72 commit.

Regarding "request", I've left a comment here: https://github.com/kubernetes/website/pull/18468#discussion_r365350517

Regarding "scheduler", I've left a comment for discussion here:
https://github.com/kubernetes/website/pull/18468#discussion_r365351689

And here's some worth some discussion:
https://github.com/kubernetes/website/pull/18468#discussion_r365354586
https://github.com/kubernetes/website/pull/18468#discussion_r365355424

irvifa commented:
@phanama Thanks for link you mentioned, can you please put this in our docs? To make it easier for us when we tried to translate something.

irvifa commented:
/lgtm
/approve
